doc/control-tower.md

```diff
@@ -0,0 +1,160 @@
+
+If you need to setup these resources in your AWS environment and get a managed AWS service to govern your AWS Organization and account structure, Control Tower is the recommended solution.
+
+If you already bootstraped your landing zone using the bootstrap kit, this document will guide you through the "migration" to Control Tower.
+
+At the end of this guide, you will have what bootstrap kit created: the Organization, OUs (Shared, SDLC, Prod), accounts (CICD, Dev, Staging, Prod, ...) plus what Control Tower will create (see above).
+
+## Prerequisites
+
+**Disclaimer**: This page is only needed when you already have used bootstrap kit to setup your environment. If you start from a blank page or an existing Organization not created by bootstrap kit, refer to the Control Tower documentation.
+
+When you setup a landing zone with Control Tower, it performs some pre-check (see [doc](https://docs.aws.amazon.com/controltower/latest/userguide/getting-started-with-control-tower.html)):
+
+### CloudTrail integration
+
+One of them is about AWS CloudTrail. CloudTrail must not be enabled for your organization. Otherwise you will get the fllowing warning:
+
+![You must unsubscribe your organization from AWS CloudTrail](migration-ct-error-cloudtrail.png)
+
+In order to do so, you can use the console or the CLI:
+
+- Using the console, in AWS Organizations > Services > CloudTrail, click on **Disable trusted access** button:
+
+![Disable trusted access for CloudTrail](migration-ct-org-cloudtrail.png)
+
+- Using the CLI, type the following command with the Administrator user on the management account:
+
+```shell
+$ aws organizations disable-aws-service-access --service-principal cloudtrail.amazonaws.com
+```
+
+Note: The CloudTrail trail is not removed, and your CloudTrail logs are not deleted.
+
+### Config integration
+
+Another pre-requisite, which is not checked by Control Tower, but only recommended ([source](https://docs.aws.amazon.com/controltower/latest/userguide/enroll-account.html#enrollment-prerequisites)):
+> We recommend that the account should not have an AWS Config configuration recorder or delivery channel. These may be deleted through the AWS CLI before you can enroll an account. Otherwise, see Enroll accounts that have existing AWS Config resources (https://docs.aws.amazon.com/controltower/latest/userguide/existing-config-resources.html) for instructions on how you can modify your existing resources.
+
+If you prefer not to delete these resources, skip these 2 steps and follow the documentation (you will have to create a ticket to support).
+
+#### Configuration Recorder
+
+To remove the Configuration recorder, you can use the following commands:
+
+```shell
+$ aws configservice describe-configuration-recorders
+{
+    "ConfigurationRecorders": [
+        {
+            "name": "BlueprintConfigRecorder",
+            "roleARN": "arn:aws:iam::123456789012:role/Prod-orgStack-SecureRootUserConfigRecorderConfigRe-ZA31CP5DNHZ6",
```

Comment:
We should mention that they need to replace `123456789012` by their account id.

Reply:
this is actually the result of the command, but I will describe it.